tx/textual/valuerenderer/any.go

```diff
@@ -93,7 +87,7 @@ func (ar anyValueRenderer) Parse(ctx context.Context, screens []Screen) (protore
 		return nilValue, err
 	}
 
-	anyMsg, err := anypb.New(internalMsg.Message().Interface())
+	anyMsg, err := any.New(internalMsg.Message().Interface())
```

Comment:
I also updated in this PR to use our own Any (without the google prefix). It's slightly orthogonal to this Spec PR, but decided to include it here since it touches the same `any.json` file.

I can separate into 2 PRs if it's confusing.